docs/contributor_guide/contributing_documentation.rst

```diff
@@ -94,6 +94,10 @@ ensure that they all render properly.
     pull request) and need help, simply @mention
     :code:`@fairlearn/fairlearn-maintainers` to get help.
 
+Code Style
+^^^^^^^^^^
+View the :ref:`code_style` section for an overview of the style we expect for contributed
+documentation files.
```

Comment:
I am thinking if maybe the documentation style paragraph would more naturally fit here instead of redirecting folks out of the doc? We should probably think as a more of a long-term goal to rethink how the information is spread out.

Reply:
Yes, I think so too. I will move this content.

Reply:
Done!